docs/book/src/reference/versions.md

```diff
@@ -120,12 +120,15 @@ Cluster API maintainers might decide to support API versions longer than what is
 
 | API Version | Status         | Supported Until                                                                  |
 |-------------|----------------|----------------------------------------------------------------------------------|
-| v1beta1     | Supported      | at least 9 months or 3 minor releases after a newer API version will be released |
-| v1alpha4    | Not served (*) | EOL since 2023-12-05 - v1.6.0 release date                                       |
-| v1alpha3    | Not served (*) | EOL since 2023-07-25 - v1.5.0 release date                                       |
+| v1beta2     | Supported      | at least 9 months or 3 minor releases after a newer API version will be released |
+| v1beta1     | Deprecated     | Deprecated since CAPI v1.11, EOL planned for v1.14, Aug 26                       |
+| v1alpha4    | Not served (*) | EOL since 2023-12-05 - v1.6.0 release date; removal planned for v1.13, Apr 26    |
+| v1alpha3    | Not served (*) | EOL since 2023-07-25 - v1.5.0 release date; removal planned for v1.13, Apr 26    |
 
-(*) Cluster API stopped to serve v1alpha3 API types from the v1.5 release and v1alpha4 types starting from the v1.6 release. 
-Those types still exist in Cluster API while we work to a fix (or a workaround) for [10051](https://github.com/kubernetes-sigs/cluster-api/issues/10051). 
+See [11920](https://github.com/kubernetes-sigs/cluster-api/issues/11920) for details about the v1beta14 removal plan.
+See [11919](https://github.com/kubernetes-sigs/cluster-api/issues/11919) for details about the v1alpha3/v1alpha4 removal plan.
+- (*) Cluster API stopped to serve v1alpha3 API types from the v1.5 release and v1alpha4 types starting from the v1.6 release.
```

Comment:
Making this a list looks a bit awkward

![image](https://github.com/user-attachments/assets/4e0380e2-f38c-4521-b48b-ce68d38f4f31)

But feel free to keep it if you prefer